src/Compilers/VisualBasic/Portable/Syntax/SyntaxKindFacts.vb

```diff
@@ -18,7 +18,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' Determine if the kind represents a contextual keyword
         ''' </summary>
         Public Shared Function IsContextualKeyword(kind As SyntaxKind) As Boolean
-            Return kind = SyntaxKind.ReferenceKeyword OrElse
+            Return kind = SyntaxKind.LoadKeyword OrElse kind = SyntaxKind.ReferenceKeyword OrElse
```

Comment:
I don't think either of these should be contextual keywords (they're just preprocessor keywords).